content/docs/06-integrations/00-prometheus-operator.md

```diff
@@ -36,6 +36,165 @@ We recommend you use version v44.3.x or greater moving forward for a simplified
+
+If you do not enable the ingress option, then by default a service with a load balancer will be created that exposes port 80. 
+
+
+Toggle the **Enable** button to enable the use of Ingress.
+
+<br />
+
+
+#### Thanos SideCar
+
+[Thanos](https://prometheus-operator.dev/docs/operator/thanos/) is an open-source system for running large-scale, distributed, and highly available Prometheus setups. Thanos allows Prometheus to store data for extended periods in object storage, such as Amazon S3 or Google Cloud Storage, instead of a local disk. This enables Prometheus to scale horizontally without the risk of using up local storage space.
+
+
+Toggle the **Enable** button to enable the use of Thanos.
+
+<br />
+
+#### Object Store
+
+Select the Thanos object storage type you will use. Review the `thanos.objstoreConfig` parameters to configure the use of object storage with Thanos. Refer to the [Thanos Object Storage](https://github.com/thanos-io/thanos/blob/main/docs/storage.md) documentation to learn more about how to configure each object storage.
+
+<br />
+
+```yaml
+charts:
+  kube-prometheus-stack:
+    prometheus:
+      prometheusSpec:
+        thanos:
+          objstoreConfig:
+```
+
+#### Thanos Ruler Object Store
+
+By default, Thanos Ruler event data is saved in object storage specified for Thanos, but you can specify a different object storage for event data.  Refer to the [Thanos Ruler](https://prometheus-operator.dev/docs/operator/thanos/?#thanos-ruler) resource to learn more.
+
+
+<br />
+
+#### Remote Monitoring
+
+You can configure the Prometheus server to accept metrics from Prometheus agents and become a centralized aggregation point for all Kubernetes metrics. Enabling this feature will expose port 9090 of the *prometheus-operator-prometheus* service. Use the generated service URL to provide other Kubernetes clusters with the [Prometheus Agent](/integrations/prometheus-agent) installed so that cluster metrics can be forwarded to the Prometheus server. 
+
+
+The remote monitoring feature is configured with defaults to help you consume this feature out-of-the-box. You can change any configuration related to remote monitoring to fine-tune settings for your environment.
+
+Refer to the [Prometheus Remote Write](https://prometheus.io/docs/practices/remote_write/) resource to learn more about configuration options.
+
+To get started with remote monitoring, check out the [Deploy Monitoring Stack](/clusters/cluster-management/monitoring/deploy-monitor-stack) guide.
+
+<br />
+
+#### Palette Resources Monitoring
+
+You can access internal Palette metrics in Grafana by adding the [Prometheus Cluster Metrics](/integrations/prometheus-cluster-metrics) pack to all your client clusters. Refer to the [Enable Monitoring on Host Cluster](/clusters/cluster-management/monitoring/deploy-agent/) guide to learn more.

```

Comment:
@karl-cardenas-coding [Prometheus Cluster Metrics](/integrations/prometheus-cluster-metrics) seems to be a broken link.

Reply:
Ah, the extra `/` at the end. Nice catch @ritawatson.